.storybook/pages/CoursePlannerEdit/CoursePlannerEdit.tsx

```diff
@@ -473,10 +473,10 @@ export const CoursePlannerEdit = () => {
 
   return (
     <>
-      <Panel className="u-margin-bottom-lg" variant="squared">
+      <Panel className="mb-6" variant="squared">
         <LayoutContainer>
           <PageHeader
-            className="u-margin-bottom-none"
+            className="mb-0"
```

Comment:
I think this one needs to be `!important`. In [the chromatic page for this story](https://www.chromatic.com/test?appId=61313967cde49b003ae2a860&id=6334b5ea26cfcb769c19bfd4), this page header is much taller than before
<img width="1632" alt="" src="https://user-images.githubusercontent.com/7761701/193079449-1c612951-6803-42de-ab88-094ad7ab2a13.png">

When I inspect the page header, I see this class is being overridden by the default margin style.
<img width="1759" alt="" src="https://user-images.githubusercontent.com/7761701/193079432-731fd695-6627-479c-8db2-c798b7ee9322.png">

(To get to the chromatic changes, click on the "Details" button next to the test that says there are changes that need to be accepted.)
<img width="838" alt="" src="https://user-images.githubusercontent.com/7761701/193079391-e13df0e5-729a-4d98-a04d-94c8ff6c0d7a.png">